src/commands/game_commands/play.ts

```diff
@@ -210,24 +209,24 @@ export default class PlayCommand implements BaseCommand {
 
     call = async ({
         message,
-        gameSessions,
         parsedMessage,
         channel,
     }: CommandArgs): Promise<void> => {
-        const guildPreference = await getGuildPreference(message.guildID);
-        const voiceChannel = getUserVoiceChannel(
-            MessageContext.fromMessage(message)
-        );
+        const messageContext = MessageContext.fromMessage(message);
+        const guildID = message.guildID;
+        const existingGameSession = Session.getSession(guildID) as GameSession;
```

Comment:
The reason we use `gameSessions[message.guildID]` everywhere instead of saving it to a variable is to account for race conditions where the session is deleted before the line runs. Can you make sure it's not an issue here before making this change?

Reply:
Forgot about that, I can revert these changes